[Improve][doc] add broker config for Oauth2 (#15201)
apache/pulsar: site2/docs/security-oauth2.md

```
> Note: The support for OAuth2 authentication is only available in Node.js client 1.6.2 and later versions.

## Broker configuration

> To configure brokers to authenticate clients, add the following parameters to broker.conf or standalone.conf:

Reviewer: ```suggestion
To enable OAuth2 brokers authentication in brokers, add the following parameters to the `broker.conf` or `standalone.conf` file.
```

> To enable OAuth2 brokers authentication in brokers, add the following parameters to the `broker.conf` or `standalone.conf` file.

Reviewer: `brokers` can be removed between `OAuth2` and `authentication`?

Author: yes, I think so, and this is a suggestion given by your. 
My fault!  I did not review it carefully.

```
> Note: The support for OAuth2 authentication is only available in Node.js client 1.6.2 and later versions.

## Broker configuration
To configure brokers to authenticate clients, add the following parameters to broker.conf or standalone.conf:
```properties

> # Configuration to enable authentication and authorization

Reviewer 2: Update this description.

> # Configuration to enable authentication and authorization

Reviewer 2: ```suggestion
# Configuration to enable authentication
```

authenticationEnabled=true
authenticationProviders=org.apache.pulsar.broker.authentication.AuthenticationProviderToken
tokenPublicKey=/path/to/publicKey

> superUserRoles=[my-super-user-1,my-super-user-2]

Reviewer 2: This also should be removed, it is authorization config.

> superUserRoles=[my-super-user-1,my-super-user-2]

Reviewer 2: ```suggestion

```

```
> Note: The support for OAuth2 authentication is only available in Node.js client 1.6.2 and later versions.

## Broker configuration
To configure brokers to authenticate clients, add the following parameters to broker.conf or standalone.conf:
```properties
# Configuration to enable authentication and authorization
authenticationEnabled=true

> authorizationEnabled=true

Reviewer 2: I think should be `authorizationEnabled=true` removed.

Reviewer 2: You should remove any authorization config like `superUserRoles`, and update the description about authentication.

Reviewer 2: @liangyepianzhou
